tests/unit_test/data_processor/history_test.py

```
@@ -97,6 +134,18 @@ def test_delete_user_conversations(self, mock_history):
                                                    till_date_timestamp=till_date_timestamp)
         assert True
 
+    @mock.patch('kairon.history.processor.HistoryProcessor.delete_user_conversations', autospec=True)
+    def test_delete_user_conversations_till_date(self, mock_history):
```

Ruff (0.14.3)

Unused method argument: `mock_history`

(ARG002)